README.md

```diff
@@ -46,6 +54,40 @@ const options = {
 };
 <MUIDatatable columns={columns} data={data} options={options} selectCellPostion="right" />
 ```
+
```

Comment:
@JoonLeeNIH Can we create a separate readme under "src/components/OverlayWindow/"? You can name it "README.md"

Reply:
Sure. I have now added a README for this specific component